samples/oauth_pkce_cli/README.md

```diff
@@ -36,6 +36,12 @@ Options:
   -r, --redirect_uri <redirect uri>              redirect uri (default: "http://localhost:8080/redirect")
   -h, --help                                     output usage information
 ```
+If you have issues running the usage statement above run
+
+```
+npm install
```

Comment:
This is documented below (line 49)... maybe we have a new `## Install` section above `## Usage` that has it in one place?